lace/emulator/pnd_archive.py

```diff
@@ -11,362 +11,530 @@
 from lace.cosmo import fit_linP
 
 
+def get_sim_option_list(sim_suite):
+    """
+    Get the simulation option list based on the specified simulation suite.
+
+    Args:
+        sim_suite (str): Name of the simulation suite.
+
+    Returns:
+        list: List of simulation options available for the specified simulation suite.
+
+    """
+    if (
+        (sim_suite == "Pedersen21")
+        | (sim_suite == "Cabayol23")
+        | (sim_suite == "768_768")
+    ):
+        sim_option_list = [
+            "h",
+            "nu",
+            "central",
+            "diffseed",
+            "curved",
+            "diffigm",
+            "running",
+        ]
+        sim_especial_list = sim_option_list.copy()
```

Comment:
Where is sim_special_list used?

Reply:
This is used in: 
- Line 385. We use it here for the name and folders of the files
- Line 413. Here because for especial simulations we need to recompute the linear power spectrum, while for the hypercube was stored

Reply:
Sounds good. I don't think we have done any recent change in how we parameterise the linear power spectrum, but just in case at some point (not necessarily in this PR) we should check that we are able to reproduce the old linear power stored in the files